quadriga/src/main/java/edu/asu/diging/quadriga/api/v1/model/JobPatternInfo.java

```diff
@@ -0,0 +1,38 @@
+package edu.asu.diging.quadriga.api.v1.model;
+
+/*
+ * @JobPatternInfo The object of this class is returned when @MapGraphToTripleController is called to map patterns in a collection.
+ * @param jobId is the jobId of the job created for processing the pattern.
+ * @param track is the URI of the status of the job.
+ * @param explore is the URI of the collection whose graphs are being mapped with the input patterns.
```

Comment:
I don't think class javadoc as params. Variable explanations should go with the variables or their getters/setters.